### PR TITLE
Add information to services

### PR DESCRIPTION
**Pull request by nessy**
**Related issues**
- https://myparcelcombv.atlassian.net/browse/MP-2755

### CODE REVIEW

File: specification/schemas/Service.json [hunk 1]
```diff
@@ -33,6 +33,16 @@
...
             },
+            "information": {
+              "type": "array",
+              "example": [
+                {
+                  "type": "notice",
+                  "text": "Volumetric weight may be applied."
+                }
+              ],
+              "description": "Additional information about the service"
```
Comment (NickVries) [inline specification/schemas/Service.json]:
```suggestion
              "description": "Additional information about the service."
```

File: specification/schemas/Service.json [hunk 2]
```diff
@@ -33,6 +33,16 @@
               "example": "spring-pap",
               "description": "Slug of the service"
             },
+            "information": {
+              "type": "array",
+              "example": [
+                {
+                  "type": "notice",
+                  "text": "Volumetric weight may be applied."
```
Comment (NickVries) [inline specification/schemas/Service.json]: I would maybe call this `body` instead of `text`.

File: specification/schemas/Service.json [hunk 3]
```diff
@@ -33,6 +33,16 @@
               "example": "spring-pap",
               "description": "Slug of the service"
             },
+            "information": {
+              "type": "array",
+              "example": [
+                {
+                  "type": "notice",
```
Comment (NickVries) [inline specification/schemas/Service.json]: Do you have an idea on how many different `types` there will be? Maybe we can make this an enum?

Comment (nessy) [reply specification/schemas/Service.json]: We don't know the rest of the types, so we will just add them later

**Pull request merged**